readme.md

```diff
@@ -1,104 +1,175 @@
-![Metrics Logo](docs/images/metrics_logo.png)
+# Getting Started
 
-Metrics is a set of software components to collect and review software project metrics like performance, build stability, and codebase quality.
+Consider this useful links that may help you to get started:
+1. [GitHub Agile process :chart_with_upwards_trend:](docs/02_process.md)
+2. [Dart code style :nail_care:](docs/09_dart_code_style.md)
+3. [Collaboration :raised_hands:](docs/10_collaboration.md)
```

Comment:
Would it be useful to add links to the `Effective Dart` and `process` document here?